R/dindep.R

```diff
@@ -0,0 +1,30 @@
+#' @title density of the independence copula
```

Comment:
Should use title case.

Reply:
Sorry, I meant title case as in "Density of the Independence Copula" (capital D, I, and C).